src/rprblender/export/light.py

```diff
@@ -45,6 +45,7 @@ def get_radiant_power(light: bpy.types.Light, area=0.0):
     if light.type in ('POINT', 'SPOT'):
         units = rpr.intensity_units_point
         if units == 'DEFAULT':
+            default_intensity = color * math.sqrt(light.energy) # to match cycles
```

Comment:
Way too dark if combined with the next line. Use
return color * math.sqrt(light.energy)
instead and remove the next line.